doc/classes/OS.xml

```diff
@@ -710,6 +712,9 @@
 			</argument>
 			<description>
 				Sets whether IME input mode should be enabled.
+				If active IME handles key events before the application and creates an composition string and suggestion list.
+				Application can retrieve the composition status by using [code]get_ime_selection[/code] and [code]get_ime_text[/code] functions.
```

Comment:
Use `[method method_name]` instead of `[code]` to generate hyperlinks to methods.

Reply:
Done.